### PR TITLE
Jc/add show space to method options

### PR DESCRIPTION
**Pull request by jeancarson**
# Why?
Addressing [this](https://github.com/intercom/intercom/issues/328926) issue.
The showSpace option wasn't included in the method options on Google Tag Manager.

# How?
Following the pattern of the existing method options, I added `showSpace` to the template. 
Following the steps from these [instructions](https://developers.google.com/tag-platform/tag-manager/templates/gallery#update_your_template) to release the google tag template update, 
Screenshot 2024-07-15 at 14 55 33

### CODE REVIEW

File: .idea/.gitignore
Comment (intercomdale) [inline .idea/.gitignore:1]: @jeancarson Looks like there's some IDE config files that were autocreated & committed, could you remove those please?

Comment (jeancarson) [reply .idea/.gitignore:1]: Should be resolved, thanks

**Comment by intercomdale** [comment]
Would also suggest adding the same info as [Dzmitry's PR](https://github.com/intercom/official-gtm-tag/pull/8) so that future devs can see how to update this

**Comment by intercomdale** [comment]
Just seen that you removed the metadata.yml changes, not sure that _needs_ to be on a new commit - not suggested by this [documentation](https://developers.google.com/tag-platform/tag-manager/templates/gallery#update_your_template)

File: metadata.yaml [hunk 1]
```diff
@@ -2,9 +2,11 @@ homepage: "https://www.intercom.com/"
 documentation: "https://developers.intercom.com/installing-intercom/docs/intercom-javascript"
 versions:
   # Latest version
-  - sha: f2fefe0e0da8d34c408a22d97823740989e83ecc
+  - sha: 9f48186e15d208adc41cd095cb4e5faceb5cc356
     changeNotes: Add installation_type attribute in the IntercomSettings
   # Older versions
+    - sha: f2fefe0e0da8d34c408a22d97823740989e83ecc
```
Comment (dkremez) [inline metadata.yaml:8]: Look like line 8 and 9 are missaligned.

Comment (jeancarson) [reply metadata.yaml]: Thanks for spotting this, changed now

**Review by dkremez — COMMENTED** [review]
Sure, one small misalignment in metadata.yaml file. Everything else looks good.

It might worth to see if we can add test for a new method. Just to make sure that the next person that will work on the template doesn't broke your work.

File: template.tpl [hunk 2]
```diff
@@ -267,6 +271,18 @@ ___TEMPLATE_PARAMETERS___
             "type": "EQUALS"
           }
         ]
+      },
+      {
+        "type": "LABEL",
+        "name": "showSpace_label",
+        "displayName": "If you would like to open particular Messenger space.",
```
Comment (intercomdale) [inline template.tpl:278]: I think this is missing an "a" 😄 `If you would like to open a particular Messenger space.`

Comment (jeancarson) [reply template.tpl]: Yep, changed now, thanks!

**Pull request merged**